src/Form/Type/ChoiceFieldMaskType.php

```diff
@@ -27,14 +27,14 @@ public function buildView(FormView $view, FormInterface $form, array $options)
     {
         $sanitizedMap = [];
         foreach ($options['map'] as $value => $fieldNames) {
-            foreach ($fieldNames as $fieldName) {
-                $sanitizedMap[$value][] =
-                    str_replace(['__', '.'], ['____', '__'], $fieldName);
+            if (is_array($fieldNames)) {
+                foreach ($fieldNames as $fieldName) {
+                    $sanitizedMap[$value][] = str_replace(['__', '.'], ['____', '__'], $fieldName);
```

Comment:
wouldn't it be better to just add the value on the `allFieldNames` here directly, and then only do a `array_unique` after?